drivers/vulkan/vulkan_context.cpp

```diff
@@ -2062,6 +2087,15 @@ String VulkanContext::get_device_pipeline_cache_uuid() const {
 	return pipeline_cache_id;
 }
 
+DisplayServer::VSyncMode VulkanContext::get_vsync_mode(DisplayServer::WindowID p_window) const {
+	return windows[p_window].vsync_mode;
```

Comment:
Are we sure that `p_window` will always be a valid index? Otherwise this could crash and would need an `ERR_FAIL_COND` or `ERR_FAIL_INDEX` check to guard it.

Reply:
Done, it throws an error and prints out a message when trying to set/get the VSync mode of a non-existent window.